x/oracle/keeper/result.go

```diff
@@ -137,7 +155,10 @@ func (k Keeper) SaveResult(
 		)
 
 		if err := k.channelKeeper.SendPacket(ctx, channelCap, packet); err != nil {
-			panic(err)
+			ctx.EventManager().EmitEvent(sdk.NewEvent(
+				types.EventTypeSendPacketFail,
+				sdk.NewAttribute(types.AttributeKeyReason, err.Error()),
```

Comment:
Is it better if we put error message like this?

```suggestion
				sdk.NewAttribute(types.AttributeKeyReason, fmt.Sprintf("Unable to send packet: %s", err)),
```